xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1484,7 +1484,9 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
 
     stream->source = STREAM_SOURCE_DEMUX;
     stream->pPrivate = pStream;
-    stream->flags = (CDemuxStream::EFlags)pStream->disposition;
```

Comment:
just looked at the definition of the AV_DISPOSITION flags, right, didn't knew that the demuxstream EFlags values are already a translation of AV_DISPOSITION, so you're right, the |= is false.